l3backend/l3backend-pdf.dtx

```diff
@@ -768,6 +768,47 @@
 % \end{macro}
 % \end{macro}
 %
+% \subsubsection{PDF page size (mediabox)}
+% This code is more or less a copy of the code in graphics-def.
+% The code tests that the dimensions are larger than
+% zero as setting a zero page size doesn't make sense.
+% The page size is set in \cs{@kernel@before@begindocument}:
+% this is after the backend code has been loaded but before other
+% settings from package like geometry or graphics or memoir.
+% Similar to the setting in graphics nothing is done if
+% mag has been set. For dvi backends we delay the setting
+% to the shipout/firstpage hook.
+% \begin{macro}{\@@_backend_pagesize_set:nn}
+%    \begin{macrocode}
+\cs_new_protected:Npn \@@_backend_pagesize_set:nn #1#2
+  {
+    \bool_lazy_and:nnT
+      {\dim_compare_p:nNn {#1}>{0pt}} {\dim_compare_p:nNn {#2}>{0pt}}
+      {
+        \__kernel_backend_literal:e
+           {papersize=\dim_eval:n{#1},\dim_eval:n{#2}}
+      }
+  }
+
+\tl_gput_right:Nn \@kernel@before@begindocument
```

Comment:
We can't be sure that this exists: remember that the backend has to work with non-LaTeX format.s